README.md

```diff
@@ -71,13 +71,18 @@ app.listen(3000)
 
 - **ttl**
 
-the time to live in seconds; default is `0`, which means that the cache is disabled.
-Example  
+a number or a function that returns a number of the maximum time a cache entry can live in seconds; default is `0`, which means that the cache is disabled. The ttl function reveives the result of the original function as the first argument.
```

Comment:
can the function return a promise?

Reply:
No [this is where the ttl is resolved async-cache-dedupe](https://github.com/mcollina/async-cache-dedupe/blob/main/src/cache.js#L279) and it simply returns a number. If we wanted to support a promise then that would require more work in the underlying dep.

Reply:
It already states - `function that returns a number`, but I can also explicitly state promises are not supported

Reply:
I will put my prediction in a sealed envelope and open it when the next issue is opened :)